### PR TITLE
refine web studio

### PR DESCRIPTION
**Pull request by imzhenyu**
- restructure the layout
- rename the setting keys
- fix bugs about service membership
- other nits

TODO: consume too much memory in web browser

### CODE REVIEW

**Comment by imzhenyu** [comment]
@mcfatealan If you happen to get some time, please help review:-)

**Comment by mcfatealan** [comment]
hi @imzhenyu, no problem, and thanks so much for taking time correcting my mistakes.

One funny thing is that I did my thesis defense yesterday, and ah, I seemed to spend too much time answering the BIG QUESTION "what is rDSN at all".... then I wasn't able to have time to put a nice end to my presentation. It seems one of my committee members wasn't very satisfied with that and gave me a "C" as my grade. Now I'm preparing for the second round presentation as you did for your master  degree. You see, the history just repeats itself ;)

Anyway, I'll write my feedback down after learning your improvements tomorrow. Thanks for your understanding.

File: src/tools/webstudio/app_package/static/js/setting.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ var vm = new Vue({
     el: '#app',
     data:{
         setting_list: [
-            {'name':'dsn_rpc_address','label':'DSN RPC Address','description':'the machine address you use when do single-node profiling','value':''},
-            {'name':'meta_server_address','label':'Meta Server Address','description':'the meta server address for app store','value':''}
+            {'name':'target_server','label':'Server','description':'address of target machine which we care about its runtime information','value':'srgx-02:34802'},
+            {'name':'target_meta_server','label':'Meta Server','description':'address of meta server','value':'srgx-02:34602'},
+            {'name':'target_app_store','label':'App Store Server','description':'address of app store','value':'srgx-02:34602'},
```
Comment (mcfatealan) [inline src/tools/webstudio/app_package/static/js/setting.js:7]: why app store address should be configured? Do you mean we should decouple app store and profiling display?

**Comment by mcfatealan** [comment]
hi @imzhenyu, I've already learned your modification, seems good to me, thx!

File: src/tools/webstudio/app_package/static/js/setting.js (hunk 1 above)
Comment (imzhenyu) [inline src/tools/webstudio/app_package/static/js/setting.js:7]: app store and meta server are not necessarily the same service. we can deploy on store with many meta servers.

**Comment by imzhenyu** [comment]
@mcfatealan thanks, and good luck for your thesis defense!

File: src/tools/webstudio/app_package/static/js/setting.js (hunk 1 above)
Comment (mcfatealan) [inline src/tools/webstudio/app_package/static/js/setting.js:7]: yes, but request to app store by default goes to the website host. Like web studio running on "http://srgsi-11:8088/", then that would be the app store address. No need to leave such a setting block.

**Comment by mcfatealan** [comment]
@imzhenyu thx! Good luck to you too!

File: src/tools/webstudio/app_package/static/js/setting.js (hunk 1 above)
Comment (imzhenyu) [inline src/tools/webstudio/app_package/static/js/setting.js:7]: You are right - I'll remove that later on.

**Pull request merged**